### DISCUSSION TITLE
Xiaomi Temp Humidity Sensors will not re-pair with Z2M after Zigbee Stick Hardware change

### DISCUSSION THREAD

**Discussion by eddomak** [devbis/z03mmc#124] [Q&A]
Hi, I had 10 Xiaomi Temp & Humidity sensors converted to ZB and successfully paired with Z2M using Conbee II hardware. I migrated to Sonos ZB Dongle P and managed to re-pair all devices (easily) except the Temp & Humidity Sensors which will not pair at all. I tried new battery, joining the terminals, etc, all without any pairing or errors.

I then took a fresh/new Temp & Humidiity sensor out of the box and flashed the same firmware and it paired with Z2M.

Please can anyone help with how I can get those 10 working again? As I can't pair via Z2M I can't do the firmware update either. Thanks.

**Comment by devbis** [comment]
This problem was fixed in version 1.0.5
Maybe you still can upgrade fw with Conbee II with OTA?
If you cannot update over the air, the only way is to flash with uart and wires

**Reply by eddomak** [reply]
Thanks for the quick answer. Strange thing is that I had only flashed7 of  them on 5th December 2023 (using transitional firmware method), so by then 1.0.6 was out (and I am pretty sure I remember seeing some of the features) remaining 3 devices I used the PVVX branch to flash only a  couple of weeks ago. Problem is that none of them from either the first batch or the 2nd batch connect. :( 

I guess I will try by restoring Home Assistant from full (pre-migration) backup with Conbee hardware, flash OTA, then restore to a present day backup with Sonoff hardware...
